app/screens/WelcomeScreen.js

```diff
@@ -8,8 +8,19 @@ function WelcomeScreen({navigation}) {
             <Text style={styles.text}>
               Welcome to the Welcome screen!!
             </Text>
+            <View style={styles.imagecontainer}>
+                <Image
+                    style={styles.logo}
+                    source={{
+                    uri: 'https://educast.library.gatech.edu/wp-content/uploads/2020/10/cropped-logoGearsOnlyRound-1-2.png'}}
+                />
+            </View>
             <TouchableOpacity onPress={() => navigation.navigate('Home')} >
-              <Text>Click me!</Text>
+                <Image
+                    style={styles.button}
+                    source={{
+                    uri: 'https://i.stack.imgur.com/4G1qY.png'}}
```

Comment:
Could we upload the file directly into repository instead of the internet please?